api/v1/flow/flow.proto

```diff
@@ -96,16 +96,17 @@ enum TraceObservationPoint {
     TO_HOST = 2;
     TO_STACK = 3;
     TO_OVERLAY = 4;
+    TO_NETWORK = 5;
```

Comment:
Independently of the other discussion about the `cilium monitor` compatibility, we do have a strict policy to not break the Hubble gRPC API, and this is unfortunately a breaking change that would break all existing Hubble clients.

Please assign sequence number 11 to `TO_NETWORK`.

Reply:
+1, we can't change existing values. Add a new pair at the bottom. 

You can split into two sections to/from to group visually, but new entries can only get new values.

Reply:
Done.